### PR TITLE
Implement 5 star Rating in Crypsis #981

### PR DESCRIPTION
**Pull request by xillibit**
Pull Request for Issue #981 .

### CODE REVIEW

**Comment by rich20** [comment]
- [ ] Rating stars are always displayed no matter which setting
- [ ] Click on star - Warning! [object Object]

**Comment by 810** [comment]
Notice: Trying to get property of non-object in \components\com_kunena\template\crypsis\layouts\topic\item\rating\default.php on line 16

File: src/components/com_kunena/template/crypsis/layouts/topic/item/rating/default.php [hunk 1]
```diff
@@ -12,28 +12,9 @@
...
-		<input class="kratingstar" type="radio" id="star1" name="krating" value="1" <?php if ($this->rating == 1)
-		{
-			echo ' checked="checked" ';
-		} ?>/><label for="star1" title="COM_KUNENA_TOPIC_RATING_SUCKS">1 star</label>
-		<input type="hidden" id="krating_submit_url" name="url" value="<?php echo 'index.php?option=com_kunena&view=topic&layout=rate&topicid=' . $this->topicid . '&format=raw'; ?>" />
-	</form>
-</div>
+
+<?php if ($this->category->allow_ratings): ?>
+<input id="topic_id" type="hidden" value="<?php echo $topic->id ?>" /">
```
Comment (810) [inline src/components/com_kunena/template/crypsis/layouts/topic/item/rating/default.php]: change to `$this->topicid`

**Comment by xillibit** [comment]
Try now

**Pull request merged**